packages/atlas/lib/src/marker.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:meta/meta.dart';
+import 'package:atlas/atlas.dart';
+
+/// Marks a geographical location on the map.
+class Marker {
+  /// Uniquely identifies a `Marker`.
+  final String id;
+
+  /// The location where the `Marker` is drawn is represented as a `CoordinatePair`.
+  final CoordinatePair position;
```

Comment:
I would keep the variable named as the class.. `final Coordinate coordinate`

Reply:
Renamed to `LatLng` 👍